app/static/js/volunteerProfile.js

```diff
@@ -114,6 +114,24 @@ $(document).ready(function(){
       }
     })
   });
+  $("#updatePhone").click(function(){
+    userName=$(this).data("username") 
+    phoneNumber=$("#newinput").val()
+    let isvalid =phoneNumber.replace(/\D/g,"").length
```

Comment:
isValid should be boolean, not the length. So either you do the check on this line and keep it as `isValid`, or you change your variable name